core/src/main/java/org/bitcoinj/core/Message.java

```diff
@@ -138,10 +136,6 @@ public boolean isCached() {
         return payload != null;
     }
 
-    public boolean isRecached() {
```

Comment:
Maybe we would like to deprecate this for one release cycle, rather than remove it immediately?

Reply:
In general, we try to deprecate for one cycle.

I merged the `hasMoreBytes` removal without deprecation because I consider it unlikely that someone has extended the message framework *and* has used this method. I may be wrong, so if someone complains we can re-add it back in deprecated form.